src/comps/forms/PostImageModal.vue

```diff
@@ -38,7 +39,20 @@ export default {
     active: {
       type: Boolean,
       default: false
+    },
+    inputValue: {
+      type: String,
+      default: ''
     }
+  },
+
+  data: () => ({
+    postPicture: ''
+  }),
+
+  created () {
+    this.postPicture = this.inputValue
   }
+
```

Comment:
Acho que isso aqui é suficiente:
```suggestion
    modelValue: {
      type: String,
      default: ''
    }
  }
```

Reply:
eu testaria antes de aprovar o commit kk

Reply:
nice, done 8ce69aa

Reply:
nice, done 8ce69aa